tests/unit_tests/test_ecephys.py

```diff
@@ -220,3 +225,43 @@ def test_check_spike_times_not_in_unobserved_interval_multiple_units():
         object_name="TestUnits",
         location="/",
     )
+
+
+@pytest.mark.skipif(get_package_version(name="pynwb") < version.Version("2.1.0"))
+def test_check_optional_columns_electrode_table_pass():
+    electrode_table = ElectrodeTable(name="electrodes")
+    electrode_table.add_row(
+        location="unknown",
+        group=ElectrodeGroup(name="test_group", description="", device=Device(name="test_device"), location="unknown"),
+        group_name="test_group",
+    )
+    assert check_optional_columns_electrode_table(electrode_table=electrode_table) is None
+
+
+@pytest.mark.skip  # TODO: when fixed on PyNWB side, unskip
+def test_check_optional_columns_electrode_table_fail():
+    electrode_table = ElectrodeTable(name="electrodes")
+    electrode_table.add_row(
+        x=np.nan,
+        y=np.nan,
+        z=np.nan,
+        imp=np.nan,
+        location="unknown",
+        filtering="unknown",
+        group=ElectrodeGroup(name="test_group", description="", device=Device(name="test_device"), location="unknown"),
+        group_name="test_group",
+    )
+    if get_package_version(name="pynwb") >= version.Version("2.1.0"):
```

Comment:
I think this logic should go inside the skip statement. From the comments it seems like maybe you agree?

Reply:
This one is temporary until they fix the bug - this PR will have to stay in draft until then.